docs/topics/dokkatoo-migration.md

```diff
@@ -0,0 +1,359 @@
+  
+    ```kotlin
+    remoteUrl.set(URI("https://github.com/your-repo"))  
+    ```
+
+* **Custom assets:** The [`customAssets`](dokka-html.md#customize-assets) property now uses collections of files (`FileCollection`) instead of lists (`var List<File>`).
+
+  Previous configuration:
+
+    ```kotlin
+    customAssets = listOf(file("example.png"), file("example2.png"))   
+    ```
+
+  New configuration:
+
+    ```kotlin
+    customAssets.from("example.png", "example2.png")
+    ```
+
+### Share Dokka configuration across modules
+
+DPG v2 moves away from using `subprojects {}` or `allprojects {}` to share configuration across modules. In future Gradle versions, 
+using these approaches will lead to errors. For more information, see [Gradle's documentation about isolated projects](https://docs.gradle.org/current/userguide/isolated_projects.html).
+
+Follow the steps below to properly share Dokka configuration in multi-module projects with DPG v2.
+
+> For a multi-module project example, see the [Dokka GitHub repository](https://github.com/Kotlin/dokka/tree/master/examples/gradle/dokka-multimodule-example).
+>
+{type="tip"}
+
+#### Multi-module projects without convention plugins
+
+1. In your project root, create a `buildSrc` directory containing two files:
+
+   * `settings.gradle.kts`
+   * `build.gradle.kts`
+   
+2. In the `buildSrc/settings.gradle.kts` file, add the following snippet:
+   
+   ```kotlin
+   rootProject.name = "buildSrc"
+   ```
+   
+3. In the `buildSrc/build.gradle.kts` file, add the following snippet:
+
+    ```kotlin
+    plugins {
+        `kotlin-dsl`
+    }
+    
+    repositories {
+        mavenCentral()
+        gradlePluginPortal()
+    }
+    
+    dependencies {
+        implementation("org.jetbrains.dokka:dokka-gradle-plugin:2.0.0")
+    }   
+    ```
+   
+4. Create a `buildSrc/src/main/kotlin/dokka-convention.gradle.kts` directory to host the convention plugin.
+5. In the `dokka-convention.gradle.kts` file, add the following snippet:
+
+    ```kotlin
+    plugins {
+        id("org.jetbrains.dokka") 
+    }
+    ```
+
+   In the code above, you don't need to specify a Dokka version. The version is set in `buildSrc/build.gradle.kts`   
+
+6. Apply the Dokka convention plugin to your modules (subprojects). Add the convention plugin ID to each subproject's
+   `build.gradle.kts` file:
+
+    ```kotlin
+    plugins {
+        id("dokka-convention")
+    }
+    ```
+
+#### Multi-module projects with convention plugins
+
+1. Create a dedicated Dokka [convention plugin](https://docs.gradle.org/current/userguide/custom_plugins.html#sec:convention_plugins) that encapsulates your Dokka configuration.
+2. In the `build.gradle.kts` file of each module (subproject), apply your Dokka convention plugin:
+ 
+   ```kotlin
+   plugin {
+       id("dokka-convention")
+   }
+   ```
+
+3. Aggregate the documentation (see below) from multiple modules into a single output. Add dependencies in the root `build.gradle.kts` file
```

Comment:
Shouldn't this step also be a part of "multi-project without convention plugins"?

Reply:
Good point. @adam-enko What do you think? Should we add the aggregation step in "Multi-module projects without convention plugins"? Is it also applicable?

Reply:
I contacted you on Slack about this (: